test/unit/model/ResultsServiceTest.php

```diff
@@ -345,6 +349,7 @@ public function filterDataProvider()
                     // 2019-06-26T00:00:00+00:00
                     ResultsService::FILTER_END_TO => '1561507200',
                 ],
+                'delivery_execution_id',
```

Comment:
Is it possible to add constant to avoid those hardcoded strings?

Reply:
It is a test. I don't think it's useful to add a constant here.